server/bin/pbench-verify-indexed.py

```diff
@@ -1,52 +1,42 @@
 #!/usr/libexec/platform-python
 # -*- mode: python -*-
 
-"""Pbench Re-Index
+"""Pbench Verify Indexed
 
-Re-index all data for a given date range, "YYYY-MM-DD" to "YYYY-MM-DD".
-
-The process of re-indexing first looks to see which INDEXED, TO-INDEX-TOOL,
-and WONT-INDEX* symlinks exist in the given date range, and moves them to
-TO-RE-INDEX.
-
-NOTE: this interface is intended to be used interactively, this is NOT a
-service that runs as a cronjob.  NO RE-INDEXING STEPS SHOULD BE AUTOMATED
-AT THIS POINT.
+Review all archived tar balls and report how many have been properly indexed.
 """
-
+import logging
 import os
 import re
 import sys
-
-from pathlib import Path
 from argparse import ArgumentParser
-
-from elasticsearch1 import Elasticsearch
+from collections import defaultdict
+from pathlib import Path
 
 import pbench
+import pbench.indexer
+from elasticsearch1 import Elasticsearch, helpers
 
+_NAME_ = "pbench-verify-indexed"
 
-_NAME_ = "pbench-reindex"
-
-tb_pat_r = (
+tb_pat = re.compile(
     r"\S+_(\d\d\d\d)[._-](\d\d)[._-](\d\d)[T_](\d\d)[._:](\d\d)[._:](\d\d)\.tar\.xz"
 )
```

Comment:
There's nowhere that we can import this from (like `pbench.indexer`) so that we don't have to keep redefining it?

Reply:
Sure, there is a way to do it.  But to make the change for just one file system silly.  We'd have to use it in the other modules to make it worth doing.  I'll file separate commits here with that change.

Reply:
Cool!  You saw the two that I found and even found one more!  😀